priv/repo/migrations/20220928131105_add_execution_result.exs

```diff
@@ -6,8 +6,12 @@ defmodule Wanda.Repo.Migrations.AddExecutionResult do
       add :execution_id, :uuid, primary_key: true
       add :group_id, :uuid, null: false
       add :payload, :map, null: false
+      add :status, :string, null: false
+      add :targets, :map, null: false, default: "[]"
```

Comment:
Do we need this default value here for some reason?
It could as well be `{:array, :map}` as well, to enforce being a list.
I don't know if it works though

Reply:
please doublecheck, but i think that what we are searching for is `add :targets, :jsonb, default: "[]"`

Reply:
I followed ecto doc here https://hexdocs.pm/ecto/Ecto.Schema.html#embeds_many/3

> It is recommended to declare your [embeds_many/3](https://hexdocs.pm/ecto/Ecto.Schema.html#embeds_many/3) field with type :map in your migrations, instead of using {:array, :map}. Ecto can work with both maps and arrays as the container for embeds (and in most databases maps are represented as JSON which allows Ecto to choose what works best).

I haven't tried the `add :targets, :jsonb, default: "[]"` gonna try it

Reply:
I see, I believe this is not an `embeds_many` tho since we are not embedding schemas but we have a "free" json field

Reply:
Forgive me, i totally miss the point the you were using an embeds_many, ignore my comment above